image_width: 640
image_height: 480
camera_name: narrow_stereo
camera_matrix:
  rows: 3
  cols: 3
  data: [853.19126,   0.     , 289.59911,
           0.     , 852.31755, 281.46072,
           0.     ,   0.     ,   1.     ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.106096, 0.259542, 0.011894, -0.008502, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1., 0., 0.,
         0., 1., 0.,
         0., 0., 1.]
projection_matrix:
  rows: 3
  cols: 4
  data: [879.39343,   0.     , 287.56859,   0.     ,
           0.     , 877.7052 , 283.03407,   0.     ,
           0.     ,   0.     ,   1.     ,   0.     ]
